biocrnpyler/dna_construct.py

```diff
@@ -468,32 +432,9 @@ def changed(self):
         self.reset_stored_data()
         self.name = self.make_name()
     def update_reactions(self,norna=False):
+        return []
         
-        reactions = []
-        rnas = None
-        proteins = None
-        rnas = self.predicted_rnas
-        proteins = self.predicted_proteins
-        if((rnas is None) or (proteins is None)):
-            warn("{} is running explore_txtl possibly more than once".format(str(self)))
-            rnas,proteins = self.explore_txtl()
-        #rnas,proteins = self.explore_txtl()
-
-        if(self.out_components is None):
-            raise AttributeError("construct "+str(self) +" has no components! This happens because update_reactions has been run before update_species")
-        for part in self.out_components:
-            rx_list = []
-            #_ = part.update_species()
-            #update_components creates new components which are copies of the parts_list components
-            #but the dna_to_bind attribute has been changed.
-            #thus we need to make sure they have update_species() run on them before
-            #doing update_reactions, in case any of these parts have memory
-            reactions+= part.update_reactions()
-        for rna in proteins:
-            if(not rna == self):
-                #this part makes sure we don't do an infinite loop if we are in fact an RNA_construct
-                reactions += rna.update_reactions()
-        return reactions
+
 
 class DNA_construct(Construct,DNA):
```

Comment:
DNA construct subclasses DNA but never initializes it. Same goes for RNA construct with RNA